stonesoup/plotter.py

```diff
@@ -1696,6 +1696,7 @@ def __init__(self, dimension=Dimension.TWO, x_label: str = "$x$", y_label: str =
     def run(self,
             times_to_plot: List[datetime] = None,
             plot_item_expiry: Optional[timedelta] = None,
+            plot_title=None,
```

Comment:
I wonder if this would be better in the `__init__` method? Same as `x_label` and `y_label`.